testsuite/features/support/remote_node.rb

```diff
@@ -167,47 +167,47 @@ def wait_while_process_running(process)
     end
   end
 
-  # Copies a local file to a remote node.
+  # Copies a file from the test runner (aka controller) into the remote node.
   #
-  # @param local_file [String] The path to the file to copy.
-  # @param remote_file [String] The path in the destination.
+  # @param test_runner_file [String] The path to the file to copy.
+  # @param remote_node_file [String] The path in the destination.
   # @return [Integer] The exit code.
-  def inject(local_file, remote_file)
+  def inject(test_runner_file, remote_node_file)
     if @has_mgrctl
       tmp_folder, _code = run_local('mktemp -d')
-      tmp_file = File.join(tmp_folder.strip, File.basename(local_file))
-      code, _remote = scp(local_file, tmp_file)
-      if code.zero?
-        _out, code = run_local("mgrctl cp --user #{user} #{tmp_file} server:#{remote_file}")
+      tmp_file = File.join(tmp_folder.strip, File.basename(test_runner_file))
+      success = get_target('localhost').scp(test_runner_file, tmp_file, host: self)
```

Comment:
This command is basically the same as below in the `else` condition. I wonder if we do not need to use a `mgrctl` command like before? How does it know it is in the container? Is it due to the fact, we run this `.inject()` method on a server node object? Then the `localhost` value would make sense.

Reply:
So, on the first case, we want to copy a file from the controller to the "container" **host**, and then we move this file from the "container" **host** to inside the container.
In the second case, we copy a file from the controller to the host.

Reply:
`get_target('localhost')` will return a node that will point to our controller or test runner (if you run this in your local IDE)

Reply:
Ok, that makes sense. I somehow thought it worked the other way around. Thanks.